### DISCUSSION TITLE
Add sqlpackage capability to Ubuntu

### DISCUSSION THREAD

**Discussion by cookbr** [actions/virtual-environments#5677] [General questions]
I've got a release pipeline in Azure DevOps which is comprised of a single instance of the [SqlAzureDacpacDeployment](https://github.com/microsoft/azure-pipelines-tasks/tree/master/Tasks/SqlAzureDacpacDeploymentV1) task.  When adding this task, [a sqlpackage demand is added to the agent](https://github.com/microsoft/azure-pipelines-tasks/blob/master/Tasks/SqlAzureDacpacDeploymentV1/task.json#L23).  I've set the agent specification to ubuntu-20.04 and [confirmed that sqlpackage is installed](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md#ms-sql-server-client-tools).  When I run my pipeline, it doesn't get picked up by any agent - just hangs out in the queue indefinitely.  I'm making an assumption the problem is due to the lack of a sqlpackage capability on the type of agent I'm trying to use.  Am I overlooking/misunderstanding something or should this capability be added?

**Comment by miketimofeev** [comment]
@cookbr the sqlpackage should work out of the box. Could you create an issue in the tasks repo https://github.com/microsoft/azure-pipelines-tasks/issues
Probably there is some issue with the task
